tests/e2e/page-router.test.ts

```diff
@@ -157,12 +157,15 @@ test.describe('Simple Page Router (no basePath, no i18n)', () => {
 
         const fallbackWasServed =
           useFallback && headers1['cache-status'].includes('"Next.js"; fwd=miss')
-        expect(headers1['netlify-cache-tag']).toBe(
-          fallbackWasServed ? undefined : `_n_t_${encodeURI(pagePath).toLowerCase()}`,
-        )
+        if (!fallbackWasServed) {
+          expect(headers1['netlify-cache-tag']).toBe(`_n_t_${encodeURI(pagePath).toLowerCase()}`)
+        }
         expect(headers1['netlify-cdn-cache-control']).toBe(
           fallbackWasServed
-            ? undefined
+            ? // fallback should not be cached
+              nextVersionSatisfies('>=15.4.0-canary.95')
+              ? `private, no-cache, no-store, max-age=0, must-revalidate, durable`
+              : undefined
```

Comment:
Both of those mean "no-cache" in practice so those are generally the same in fact that fallback should not be cached

Just note that same change is applied 3 times in this PR, but will just comment on first case